packages/account/src/utils/Types.ts

```diff
@@ -1,11 +1,11 @@
 import { Signer } from "ethers";
```

Comment:
Shouldn't we use viem now ? So this should be compatible with viem client now not ethers v5

Reply:
resolved